subscription/src/main/java/org/killbill/billing/subscription/engine/dao/SubscriptionSqlDao.java

```diff
@@ -55,4 +57,10 @@ public void updateChargedThroughDate(@Bind("id") String id,
                                          @Bind("chargedThroughDate") Date chargedThroughDate,
                                          @SmartBindBean final InternalCallContext context);
 
+    @SqlUpdate
+    @Audited(ChangeType.UPDATE)
```

Comment:
FYI I've checked that we already seem to support audited batch calls: https://github.com/killbill/killbill/blob/585d68891b06f3329379548583a33fe94b9250ea/util/src/main/java/org/killbill/billing/util/entity/dao/EntitySqlDaoWrapperInvocationHandler.java#L435

Reply:
Am i missing a `@BatchChunkSize` annotation?